x/rollapp/keeper/msg_server_update_state_test.go

```diff
@@ -147,55 +146,31 @@ func (suite *RollappTestSuite) TestUpdateState() {
 		suite.Require().Nil(err)
 
 		// end block
-		responseEndBlock := suite.App.EndBlocker(suite.Ctx, abci.RequestEndBlock{Height: suite.Ctx.BlockHeight()})
+		suite.App.EndBlocker(suite.Ctx, abci.RequestEndBlock{Height: suite.Ctx.BlockHeight()})
+
+		for i := uint64(0); i <= expectedLatestStateInfoIndex.Index; i++ {
```

Comment:
calling it `expectedLatest..` an using it as a var all over the code is a bit confusing. Usually expected is used for comparison.

Reply:
changed